app/services/sign_in/assertion_validator.rb

```diff
@@ -58,6 +77,8 @@ def create_new_access_token
     end
 
     def decoded_assertion_scopes_are_defined_in_config?
+      return service_account_config.scopes.blank? if scopes.blank?
```

Comment:
can we define the scopes as an array as well? `Array(decoded_assertion.scopes)`, there's a current bug here I think, if the input scopes isn't an array I think it will crash. but using that array wrapper will force-define it as an array

Reply:
Got it, I'm putting the `Array` casting in the `scopes` definition method itself.